betterproto/tests/mocks.py

```diff
@@ -5,7 +5,7 @@
 
 class MockChannel(Channel):
     # noinspection PyMissingConstructor
-    def __init__(self, responses: List) -> None:
+    def __init__(self, responses: List = []) -> None:
```

Comment:
You can probably get away with it here but having a mutable default value is dangerous.

Usual workaround:
```python
def __init__(self, responses: Optional[List] = None) -> None:
    self.responses = [] if responses is None else responses
```

Reply:
Wow! First I thought, why would this make any difference? Then I looked it up, and I'm shocked. Thanks! Learned something 😃 for those wondering, see [here](https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)

will change this!

Reply:
pylint has a rule `dangerous-default-value` that will pick this up.

If pylint is added to this project it should be configured with allowed rules, instead of excluded ones to avoid headaches :)